### PR TITLE
Delete an unnecessay target `get-inst-libraries`

### PR DESCRIPTION
**Pull request by dalinaum**

### CODE REVIEW

**Comment by dalinaum** [comment]
Please review this @realm/java

File: realm/realm-jni/src/Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ lib_LIBRARIES = librealm-jni.a
 JNI_SOURCES := $(wildcard *.cpp)
 librealm_jni_a_SOURCES = $(JNI_SOURCES)
 
-# Used by ../../build.sh
+# Used by ../build.gradle
 get-inst-libraries:
```
Comment (zaki50) [inline realm/realm-jni/src/Makefile]: Is this target really used? It seems to be unused.
What do you think? @beeender

Comment (dalinaum) [inline realm/realm-jni/src/Makefile]: I cannot find any dependency with target `get-inst-libraries` too. Should we delete this target? @realm/java

Comment (dalinaum) [inline realm/realm-jni/src/Makefile]: I found out `calling get-inst-libraries` in delete `build.sh` file. I think it should be deleted too.

**Comment by dalinaum** [comment]
@zaki50 I think you are right. I deleted target  `get-inst-libraries` instead just changing an outdated comment.
Please review this @realm/java

**Comment by emanuelez** [comment]
Is this makefile used at all?

**Comment by zaki50** [comment]
@emanuelez I think it is used in `realm-jni/build.gradle`. That gradle file executes following external command.

```
make -C "${projectDir}/src" ...
```

**Comment by dalinaum** [comment]
@emanuelez I am not sure I quite understood your question. This Makefile include `../generic.mk`. `../build.gradle` uses targets defined in generic.mk.

**Comment by zaki50** [comment]
I think deleting `get-inst-libraries` is :+1: 
And also think that we should get another :+1: by @beeender

**Comment by beeender** [comment]
:+1:

**Comment by emanuelez** [comment]
Please be aware that both generik.mk and the Makefile were copies from core so I'm not sure it's worth to modify them.

**Comment by kneth** [comment]
I don't mind that we clean up the build files in general, but I agree with @emanuelez that it might not be worth it.

**Comment by dalinaum** [comment]
@emanuelez @kneth Although `generic.mk` file is imported from core, `Makefile` is not similar to any Makefiles of core. I think that at least our Makefile is for only Realm-java now and `get-inst-libraries` is orinallay added for Realm-java.

**Comment by beeender** [comment]
Makefile is copied from core as well, src/realm/Makefile. But that file changes a lot in core, so they don't look similar anymore :)  I think it is not harmful to remove this useless line :)

**Comment by kneth** [comment]
:+1:

**Pull request merged**